src/main/java/org/openjdk/jextract/impl/StructBuilder.java

```diff
@@ -297,10 +297,8 @@ private void emitIndexedFieldGetter(Constant vhConstant, String javaName, Class<
         append("return (" + type.getName() + ")");
         append(vhConstant.accessExpression());
         append(".get(");
-        append(seg);
-        append(".asSlice(");
-        append(index);
-        append("*sizeof()));\n");
+        append(seg + ", ");
```

Comment:
Note that in this (and the setter) indexed accessor, we can now remove slicing, as we can simply pass the offset we want to the underlying var handle. This seems like a good thing.